package.json

```diff
@@ -11,6 +11,7 @@
     "dev": "webpack-dev-server --hot --json --open --config webpack.dev.js",
     "local": "webpack-dev-server --hot --json --open --config webpack.local.js",
     "beta": "webpack-dev-server --hot --json --open --config webpack.beta.js",
+    "prod-k8": "webpack-dev-server --hot --json --open --config webpack.prod-k8.js --host 0.0.0.0",
```

Comment:
i am pretty sure `webpack-dev-server` is not for prodction use.

i believe the process is build the node app, and deploy the public folder.
you can use the ngix docker image after that: https://hub.docker.com/_/nginx